src/cms/models/events/event_translation.py

```diff
@@ -284,6 +285,30 @@ def is_up_to_date(self):
         """
         return not self.currently_in_translation and not self.is_outdated
 
+    @classmethod
+    def search(cls, region, language_slug, query):
+        """
+        Searches for all event translations and match the given `query` in their title or slug.
```

Comment:
Not really an issue, but I think we usually put an empty line between the description, parameters and return values.